sls-versions/src/main/java/com/palantir/sls/versions/OrderableSlsVersion.java

```diff
@@ -38,6 +39,36 @@ public abstract class OrderableSlsVersion extends SlsVersion implements Comparab
         SlsVersionType.RELEASE_SNAPSHOT
     };
 
+    /**
+     * The release candidate version number, if this version is a release candidate or release candidate snapshot.
+     */
+    @Value.Derived
+    OptionalInt rcVersionNumber() {
```

Comment:
nit: It feels more correct to drop the version - these are just "numbers" not "version numbers". That's also consistent with verbiage we use elsewhere.

```suggestion
    /**
     * The release candidate number, if this version is a release candidate or release candidate snapshot.
     */
    @Value.Derived
    OptionalInt rcNumber() {
```